src/timeranges/_timeranges.py

```diff
@@ -120,6 +122,8 @@ def contains(self, dt: datetime) -> bool:
         tz = self.timezone
         if tz is not None:
             dt = dt.astimezone(tz)
+        if self.day_ranges == {}:
```

Comment:
Pretty much the same as the other comment. `if not self.day_ranges` would possibly be better.